src/android/InAppBrowser.java

```diff
@@ -1446,6 +1448,46 @@ public void onReceivedError(WebView view, int errorCode, String description, Str
             }
         }
 
+        @Override
+        public void onReceivedSslError(WebView view, SslErrorHandler handler, SslError error) {
+            super.onReceivedSslError(view, handler, error);
+            try {
+                JSONObject obj = new JSONObject();
+                obj.put("type", LOAD_ERROR_EVENT);
+                obj.put("url", error.getUrl());
+                obj.put("code", 0);
```

Comment:
Upon a second review I'm thinking about actually putting the enum code of the `SslError` into the `code` field? Otherwise it's pretty useless.

Reply:
I think "code" is supposed to be http code like 404/500 etc so I defined 0 as the code because it can't be determined here.

Reply:
I was maybe wrong here about the fact that this is a http code however I still think that it should not be the enum code since it could not be distinguished from other error codes.

Reply:
Indeed, it seems in other cases this is the HTTP error code.